### PR TITLE
[Docs] Add docker-compose instructions

### PR DESCRIPTION
**Pull request by hmmbob**
### Description
Adds instructions on how to run this server using docker-compose.
Removed unused badge for "edge" branch

### CODE REVIEW

**Comment by dimaguy** [comment]
I was wondering if you could leave a note too on how do put in plugins in a shared folder with your method. Because otherwise it would be just another way to do it and thats too similar with the docker oneliner

**Comment by hmmbob** [comment]
Sure, but I don't use that option. Got some docs on how this works? Is it just mounting the folder `plugins` inside the `/app` folder? Because that's easy: one can mount any dir into the container with such compose:

```
version: '3.4'
services:
  impostor:
    image: aeonlucid/impostor:nightly
    container_name: impostor
    ports:
      - 22023:22023/udp
    volumes:
      - /path/to/plugins/on/local/pc:/app/plugins
```

**Comment by dimaguy** [comment]
Ye thats basically it. it just needs to be the ``root folder/plugins`` where impostor is on :)

**Comment by hmmbob** [comment]
Done

File: docs/Running-the-server.md [hunk 1]
```diff
@@ -18,12 +18,26 @@ There are currently two modes to run the Impostor server in. The first way is th
...
+
+services:
+  impostor:
+    image: aeonlucid/impostor:nightly
+    container_name: impostor
+    ports:
+      - 22023:22023/udp
+    volumes:
+      - /path/to/local/config.json:/app/config.json # For easy editing of the config
+      - /path/to/local/plugins:/app/plugins # Only needed if using plugins
```
Comment (reecebenson) [inline docs/Running-the-server.md:38]: Could we also add libraries, if that's the case?
`/path/to/local/libraries:/app/libraries  # Only needed if using external libraries`

Comment (hmmbob) [reply docs/Running-the-server.md]: Sure!

**Review by reecebenson — APPROVED** [review]

**Review by dimaguy — APPROVED** [review]

File: docs/Running-the-server.md [hunk 2]
```diff
@@ -18,12 +18,27 @@ There are currently two modes to run the Impostor server in. The first way is th
 ### Using docker
 
 [![Docker Image](https://img.shields.io/docker/v/aeonlucid/impostor?sort=semver)](https://hub.docker.com/repository/docker/aeonlucid/impostor)
-[![Docker Image](https://img.shields.io/docker/v/aeonlucid/impostor/edge)](https://hub.docker.com/repository/docker/aeonlucid/impostor)
```
Comment (AeonLucid) [inline docs/Running-the-server.md:21]: Why has this been removed? Please update to `https://img.shields.io/docker/v/aeonlucid/impostor/nightly`.

**Review by AeonLucid — CHANGES_REQUESTED** [review]

File: docs/Running-the-server.md (hunk 2 above)
Comment (hmmbob) [reply docs/Running-the-server.md:21]: It errored at 'edge' and the nightly was shown in the semver variant, hence why i removed it. It's back 👍

**Pull request merged**